graph.rb

```diff
@@ -19,6 +22,15 @@ def topological_sort
     end
   end
 
+  def acyclic?
```

Comment:
I know the issue calls for `acyclic?`, but I don't really like negative predicates. Maybe we can swap the booleans and call it `cyclic?` instead?